src/scss/grommet-core/_objects.search.scss

```diff
@@ -141,14 +141,31 @@
     line-height: triple($inuit-base-spacing-unit);
 
     .search__input {
-      @include inuit-font-size($large-number-font-size, double($inuit-base-spacing-unit));
-      padding: halve($inuit-base-spacing-unit - $large-number-font-size - $input-border-width)
-        ($large-number-font-size - $input-border-width);
-      padding-right: (double($inuit-base-spacing-unit) - $input-border-width);
+      @include inuit-font-size($large-number-font-size * 1.5, double($inuit-base-spacing-unit));
```

Comment:
Let's add another default setting for this instead of multiplying $large-number-font-size. We should have done that from the beginning.